scripts/build.sh

```diff
@@ -13,6 +13,7 @@ fi
 GOBUILD_TAGS="novirt noaugeas "
 VERSION=$($(dirname $0)/pkg-version --full)
 LDFLAGS="-X github.com/gluster/glusterd2/gdctx.GlusterdVersion=$VERSION"
+LDFLAGS+=" -B 0x$(head -c20 /dev/urandom | od -An -tx1 | tr -d ' \n')"
```

Comment:
@purpleidea I was also looking for this sometime back and Go [doesn't officially support](https://github.com/golang/go/issues/16860) reproducible builds yet.

@kshlm We should base build id based on git sha-1 eventually. https://fedoraproject.org/wiki/PackagingDrafts/Go#Build_ID